script.js

```diff
@@ -0,0 +1,81 @@
+function Book(title, author, pages,read) {
+    this.title = title;
+    this.author = author;
+    this.pages = pages;
+    this.read = read;
+}
+
+const myLibrary = [
+    new Book("Heroes","Joe Abercrombie", 475, false), 
+    new Book("Metamorphosis", "Franz Kafka", 189, true),
+    new Book("Hobbit", "J.R.R. Tolkien", 259, true)];
+
+function listBook(book) {
+    let card = document.createElement("div");
+    card.setAttribute("data-index", `${myLibrary.indexOf(book)}`)
+    card.classList.add("card");
+    
+    card.innerHTML = `
+    <span class="material-symbols-outlined remove-book data-index=${myLibrary.indexOf(book)}" >
+                close
+        </span> 
+    <div class="description">     
+        <h3 class="title">${book.title}</h3>
+        <p class="author">by ${book.author}</p>
+        <p>No. of pages: ${book.pages}</p>
+    </div>
+    <div class="action-buttons">     
+        <button class="read-status">${isRead(book)}</Button>
+    </div>`;
+
+    let button = card.lastChild.lastElementChild;
+    if (book.read) button.classList.add("read");
+    const displayedLibrary = document.querySelector(".library");
+    displayedLibrary.appendChild(card);
+    toggleRead(button);
+    removeBook(card);
+}
+
+function isRead(book) {
+    return book.read ? "Read" : "Not read";
```

Comment:
Loose suggestion: for bug avoiding purposes and autocomplete, it is useful to put reused values (used for checks and assignments) such as Read and Not read into enums (note that enums exist only in TS, in JS it's just an object).

```js
const BookReadStatus = { // BookReadStatus is a terrible name 😄 
  Read: 'Read',
  NotRead: 'Not read',
}
```

in TypeScript it would be:
```ts
enum BookReadStatus {
  Read,
  NotRead,
}
```

I think that enums are being pushed out of TS, but I only vaguely remember the issue, since I was out of it for some time.

Although, you can avoid bugs and get autocomplete with types also.
Something like:
```ts
type BookReadStatus = 'Read' | 'NotRead';
```